doc/examples/reconst_compare.py

```diff
@@ -0,0 +1,150 @@
+"""
+
+.. _reconst_compare:
+
+===================================================
+Compare Reconstructions with Varying Scan Qualities
+===================================================
+
+Let's explore the effect of the number of directions and b-values
+on the quality of the reconstruction.
+"""
+
+import numpy as np
+import dipy.reconst.shm as shm
+import dipy.direction.peaks as dp
+import matplotlib.pyplot as plt
+
+from dipy.denoise.localpca import mppca
+from dipy.core.gradients import gradient_table, unique_bvals_tolerance
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.io.image import load_nifti
+from dipy.segment.mask import median_otsu
+
+from dipy.denoise.localpca import mppca
+from dipy.align import affine_registration, motion_correction
+from dipy.reconst.mcsd import (auto_response_msmt,
+                               mask_for_response_msmt,
+                               response_from_mask_msmt)
+from dipy.segment.tissue import TissueClassifierHMRF
+from dipy.reconst.mcsd import MultiShellDeconvModel, multi_shell_fiber_response
+from dipy.reconst.csdeconv import (ConstrainedSphericalDeconvModel,
+                                   auto_response_ssst)
+from dipy.viz import window, actor
+
+from dipy.data import get_sphere, get_fnames
+sphere = get_sphere('symmetric724')
+
+###############################################################################
+# For this example, we use fetch to download a multi-shell dataset which was
+# kindly provided by Hansen and Jespersen (more details about the data are
+# provided in their paper [Hansen2016]_). The total size of the downloaded
+# data is 192 MBytes, however you only need to fetch it once.
+
+fraw, fbval, fbvec, t1_fname = get_fnames('cfin_multib')
+
+###############################################################################
+# Load the dMRI data.
+
+data, affine = load_nifti(fraw)
+bvals, bvecs = read_bvals_bvecs(fbval, fbvec)
+gtab = gradient_table(bvals, bvecs)
+
+###############################################################################
+# For the sake of simplicity, we only select two non-zero b-values for this
+# example.
```

Comment:
Can you add the number of b-value we have on this data and which one we select. it might not be obvious for some people when reading the code